CONTRIBUTING.md

```diff
@@ -1,62 +1,157 @@
-  <title>GitHub icon</title>
-  <path d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"/>
-</svg>
+<svg aria-labelledby="simpleicons-adobephotoshop-icon" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title id="simpleicons-adobephotoshop-icon">Adobe Photoshop icon</title><path d="M0 .3v23.4h24V.3H0zm1 1h22v21.4H1V1.3zm4.8 4.48c0-.067.14-.116.224-.116.644-.033 1.588-.05 2.578-.05 2.772 0 3.85 1.52 3.85 3.466 0 2.54-1.842 3.63-4.102 3.63-.38 0-.51-.017-.775-.017v3.842c0 .083-.033.116-.115.116H5.916c-.083 0-.115-.03-.115-.113V5.78zm1.775 5.312c.23.016.412.016.81.016 1.17 0 2.27-.412 2.27-1.996 0-1.27-.786-1.914-2.122-1.914-.396 0-.775.016-.957.033v3.864zm8.607-1.188c-.792 0-1.056.396-1.056.726 0 .363.18.61 1.237 1.155 1.568.76 2.062 1.485 2.062 2.557 0 1.6-1.22 2.46-2.87 2.46-.876 0-1.62-.183-2.05-.43-.065-.033-.08-.082-.08-.165V14.74c0-.1.048-.133.114-.084.624.413 1.352.594 2.012.594.792 0 1.122-.33 1.122-.776 0-.363-.23-.677-1.237-1.205-1.42-.68-2.014-1.37-2.014-2.527 0-1.287 1.006-2.36 2.755-2.36.86 0 1.464.132 1.794.28.082.05.1.132.1.198v1.37c0 .083-.05.133-.15.1-.444-.264-1.1-.43-1.743-.43z"/></svg>
 ```
 
-* * *
+### 6. Update The JSON Data For SimpleIcons.org
+
+Icon metadata should be added to the `_data/simple-icons.json` file. Each icon in the array has three required values:
+
+  * The `title` of the new SVG.
+  * A `hex` color value that matches the brand's primary color. All uppercase and without the `#` pound symbol.)
+  * The `source` URL of the logo being used. (Be specific!)
+
+Here is the object for the Adobe Photoshop icon as an example:
+
+```json
+{
+    "title": "Adobe Photoshop",
+    "hex": "00C8FF",
+    "source": "https://wwwimages2.adobe.com/etc/clientlibs/beagle/ace/source/font/aceui-fonts.svg"
+},
+```
+
+## Labeling Issues
+
+We use several labels to help organize and identify issues. Here's what they represent and how we use them:
+
+* icon - Issues for adding or updating an icon.
+* docs - Issues for improving or updating documentation.
+* good first issue - Issues we believe are simple and a good first stab at contributing to the project.
+* help wanted - Issues we would like help from the community to resolve.
+* awaiting reply - Issues awaiting reply from an individual (issue author or 3rd party) before it may be addressed.
```

Comment:
I really like what they did with the labels overview over at the Atom repo, see [link](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#type-of-issue-and-issue-state). Maybe we can structure it similarly 🤔 thoughts?